### ISSUE TITLE
learning forward optical flow by use softmax splatting during training

### ISSUE THREAD

**Issue by tomeramit** [sniklaus/softmax-splatting#32]
Hi,

I'm trying to learn forward optical flow using the softmax splatting (assume you have frame_0, frame_1 and you want to extract the flow that when I will do the splatting on frame_0 it will give me approximate frame_1)
but I couldnt make it work well (although when I try to use backward optical flow it's working well) 

do you have an example for using this during training? (when backpropogate through the softmax splatting)?
maybe I do something wrong, but a training example can be great :)
or any other idea why it's doesnt work

thanks

**Comment by sniklaus** [comment]
I don't have an example for your exact problem I am afraid. If you are looking for examples of how to use softmax splatting during training in general, you can search on Github for occurrences of softmax splatting like: https://github.com/visinf/multi-mono-sf
